app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,70 @@
+class PostsController < ApplicationController
+
+    # GET /posts posts.json
```

Comment:
Please remove these unnecessary comments as the code is obvious and we don't need the comments.

Reply:
sure!